packages/aws-cdk-lib/aws-cloudwatch-actions/lib/lambda.ts

```diff
@@ -0,0 +1,37 @@
+import { Construct } from 'constructs';
+import * as cloudwatch from '../../aws-cloudwatch';
+import * as iam from '../../aws-iam';
+import * as lambda from '../../aws-lambda';
+import { Stack } from '../../core';
+
+/**
+ * Use an Lambda action as an Alarm action
+ */
+export class LambdaAction implements cloudwatch.IAlarmAction {
+  private lambdaFunction: lambda.IAlias | lambda.IVersion | lambda.IFunction
+  constructor(
+    lambdaFunction: lambda.IAlias | lambda.IVersion | lambda.IFunction,
+  ) {
+    this.lambdaFunction = lambdaFunction;
+  }
+
+  /**
+   * Returns an alarm action configuration to use an Lambda action as an alarm action
+   *
+   * @see https://docs.aws.amazon.com/AmazonCloudWatch/latest/APIReference/API_PutMetricAlarm.html
+   *
+   */
+  bind(_scope: Construct, _alarm: cloudwatch.IAlarm): cloudwatch.AlarmActionConfig {
+
+    this.lambdaFunction.addPermission('AlarmPermission', {
```

Comment:
@neilkuan 

Sorry, I was only concerned about the line breaks, so it would be helpful if you could correct this one (I'll leave it as approve)

```suggestion
   * @see https://docs.aws.amazon.com/AmazonCloudWatch/latest/APIReference/API_PutMetricAlarm.html
   */
  bind(_scope: Construct, _alarm: cloudwatch.IAlarm): cloudwatch.AlarmActionConfig {
    this.lambdaFunction.addPermission('AlarmPermission', {
```

Reply:
I think linter feel this is good. XD

Reply:
Hmmm, not a lint error, but I don't see any good reason to keep it that way...

Reply:
Thanks, great!